main.go

```
@@ -1,123 +1,60 @@
 package main
 
 import (
+	"context"
 	"fmt"
-	"log"
-	"os"
-	"os/exec"
-	"path/filepath"
 
-	example "github.com/bricks-cloud/bricks/commons"
-	hclog "github.com/hashicorp/go-hclog"
-	plugin "github.com/hashicorp/go-plugin"
+	"github.com/bricks-cloud/bricks/compiler"
 )
 
-// func main() {
-// 	v, err := compiler.Build(context.Background(), "./cue/bricks", nil)
-// 	if err != nil {
-// 		fmt.Println(err)
-// 	}
-
-// 	controller := flow.New(&flow.Config{
-// 		FindHiddenTasks: true,
-// 	}, v.Cue(), ioTaskFunc)
-
-// 	if err := controller.Run(context.Background()); err != nil {
-// 		fmt.Println(err)
-// 	}
-// }
-
-// func ioTaskFunc(flowVal cue.Value) (flow.Runner, error) {
-// 	v := compiler.Wrap(flowVal)
+type Service struct {
+	Name        string
+	Description string
+	Url         string
+	Routes      []Route
+}
 
-// 	typ := v.LookupPath(cue.MakePath(
-// 		cue.Str("spotifyClient"),
-// 		cue.Str("$bricks"),
-// 		cue.Str("type"),
-// 		cue.Hid("_resourceName", "github.com/bricks-cloud/bricks/cue/bricks")),
-// 	)
+type Route struct {
+	Name        string
+	Description string
+	Paths       []string
+	Id          int
+}
 
-// 	if typ.Exists() {
-// 		fmt.Println(typ.String())
-// 	}
+type T struct {
+	Selector string
+	Type     string
+}
 
-// 	return flow.RunnerFunc(func(t *flow.Task) error {
-// 		return nil
-// 	}), nil
-// }
+// Constants to map with CUE bricks.type selectors
+const ROUTE_TYPE = "route"
+const SERVICE_TYPE = "service"
 
 func main() {
+	v, _ := compiler.Build(context.Background(), "./cue", nil)
```

let's not suppress error. if there is an error, use Panic